Src/FluentAssertions/Formatting/PredicateLambdaExpressionValueFormatter.cs

```diff
@@ -134,6 +135,8 @@ private sealed class AndOperatorChainExtractor : ExpressionVisitor
 
         public override Expression Visit(Expression node)
         {
+            Guard.ThrowIfArgumentIsNull(node);
```

Comment:
I have no idea how to construnct a test to prove the need for this check.
It doesn't seem very TDD to add code that cannot be tested :wink:

Reply:
Multiple options here:
- Disable the rule
- Guard
- `!` operator
- node from type `Expression?`

Reply:
I probably would take the last option, because this aligns with the base-class

Reply:
Another option: do nothing 🙂

Reply:
Haha :)

Yes.. I will take that option ;)